client_app/src/components/shelter/ShiftDetails.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import "../../styles/index.css";
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { TimePicker } from '@mui/x-date-pickers/TimePicker';
+import dayjs from 'dayjs';
+import dayjsUTC from 'dayjs/plugin/utc';
+import dayjsTimezone from 'dayjs/plugin/timezone';
+import ShiftDetailsTable from "./ShiftDetailsTable";
+
+dayjs.extend(dayjsUTC);
+dayjs.extend(dayjsTimezone);
+
+function ShiftDetails() {
+    const currentTime = dayjs();
+    const startHour = currentTime.add(1, 'hour').startOf('hour');
+    const endHour = startHour.add(1, 'hour');
+    return (
+      <div>
+        <div className="shift-details">
+          <div className="datetime-picker">
+            <h4>Date</h4>
+            <LocalizationProvider dateAdapter={AdapterDayjs}>
```

Comment:
The LocalizationProvider wrapper is repeated multiple times. Consider wrapping all date/time pickers within a single LocalizationProvider to avoid duplication.